### ISSUE TITLE
[MOBILE DESIGN]: GIBCT EYB: Skip to your estimated benefits link SHOULD be taller and spaced appropriately

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#14409]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a touch screen user, I want to be able to click or press the Skip to your estimated benefits link without accidentally bumping the Update benefits button.

## Details

Though the Update benefits button is **usually** disabled, I might have made some changes to the estimate your benefits form, then want to review my current benefits, and go back and update them using the skip link. Screenshot attached below.

## Acceptance Criteria

- [ ] Skip to your estimated benefits link is at least 44px tall on all devices
- [ ] Skip to your estimated benefits link has 12-14px of clear space above and below it

## Definition of done

1. Review and acknowledge feedback.
1. Update visual designs.
1. Test with mobile devices, touchscreen devices if available, and keyboard navigation.
1. Ensure no visual regression except the additional vertical space needed when the link is focused.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* https://staging.va.gov/gi-bill-comparison-tool/profile/10000008 (or any valid profile)

## Steps to Recreate

1. Enter `https://staging.va.gov/gi-bill-comparison-tool/profile/10000008` in browser
1. Tab through to the Estimate your benefits accordion. Keep tabbing until Skip to your estimated benefits link is focused. This link is directly after the Update benefits button and only appears when it has focus.
1. Verify the link is directly beneath the button.

## WCAG or Vendor Guidance (optional)

* [Medium: Accessibility and Touch Targets](https://medium.com/@zacdicko/size-matters-accessibility-and-touch-targets-56e942adc0cc)

## Screenshots or Trace Logs

![Screen Shot 2020-10-05 at 1 44 34 PM](https://user-images.githubusercontent.com/934879/95119832-3b433c80-0712-11eb-9e86-c11657dc4436.png)

**Comment by ksantiagomorgan6** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/14557

**Comment by delli-gatti-michael** [comment]
Local testing complete

![Screen Shot 2020-10-13 at 10.55.44 AM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/5a90e0b6-0619-4c0f-b4c4-ac44ee704a3b)

**Comment by 1Copenut** [comment]
@delli-gatti-michael I took a look quick. I'd like to have the yellow focus halo restored, and we should be good to close this one after.

```diff
.gi-app .profile-page .eyb-skip-link:focus {
  top: auto;
  line-height: 50px;
  margin-bottom: 1em;
  position: relative;
- outline: none;
}

**Comment by ksantiagomorgan6** [comment]
additional pr: https://github.com/department-of-veterans-affairs/vets-website/pull/14563

**Comment by 1Copenut** [comment]
@sonntag-adam Just verified this fix. Looks good and moving to close.
